picard/coverart/providers/__init__.py

```diff
@@ -134,8 +134,8 @@ def queue_images(self):
         # CoverArtProvider.WAIT
         old = getattr(self, 'queue_downloads') #compat with old plugins
         if callable(old):
-            old()
             log.warning('CoverArtProvider: queue_downloads() was replaced by queue_images()')
+            return old()
```

Comment:
This is correct, though it should be in a different PR as it is unrelated to PICARD-780.